### PR TITLE
feat(gateway): include operationName on OTel spans

### LINKED ISSUE #862: Include the `operationName` in the OpenTelemetry span attributes
As a follow-up to #836, it would be nice if the operation name which was provided by the client was attached as an OpenTelemetry span attribute.  This should be available (when provided) on the `requestContext`'s `operationName` property.  This will be an empty string (i.e., anonymous) when the operation name was not provided by the client.

### PR DESCRIPTION
**Pull request by lennyburdette**
Closes #862

### CODE REVIEW

**Review by BrynCooke — APPROVED** [review]
Lgtm

File: gateway-js/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 > The changes noted within this `vNEXT` section have not been released yet.  New PRs and commits which introduce changes should include an entry in this `vNEXT` section as part of their development.  When a release is being prepared, a new header will be (manually) created below and the appropriate changes within that release will be moved into the new section.
 
--_Nothing yet! Stay tuned_.
+- OpenTelemetry will now include the GraphQL `operationName` in span attributes, following up on the initial implementation introduced in v0.31.0 via [#836](https://github.com/apollographql/federation/pull/836)
```
Comment (abernix) [inline gateway-js/CHANGELOG.md]: @lennyburdette I've added this `CHANGELOG.md` if you want to give it a quick once-over.

**Review by abernix — APPROVED** [review]
LGTM.  Added a `CHANGELOG.md`

File: gateway-js/CHANGELOG.md (hunk 1 above)
Comment (lennyburdette) [reply gateway-js/CHANGELOG.md]: perfect, thank you!

**Pull request merged**